apps/vaults/contexts/useSolver.tsx

```diff
@@ -59,7 +61,7 @@ function	WithSolverContextApp({children}: {children: React.ReactElement}): React
 	const chainCoin = useSolverChainCoin();
 	const partnerContract = useSolverPartnerContract();
 	const internalMigration = useSolverInternalMigration();
-	const [currentSolverState, set_currentSolverState] = useState(vanilla);
+	const [currentSolverState, set_currentSolverState] = useState<TSolverContext & {hash: string}>({...vanilla, hash: ''});
```

Comment:
If the `TSolverContext` always has the need for a hash, we could just augment that type